src/utils.ts

```diff
@@ -1 +1,24 @@
+// import { dayOfWeek, monthOfYear } from './constants';
+// import { default as dayjs } from 'dayjs';
+import dayjs from 'dayjs-ext';
+
 export const toFahrenheit = (data: number): string => Math.floor((data * 9) / 5 + 32).toString();
+export const changeToDay = (date: string): any => dayjs(date).format('ddd. MMM DD, YYYY');
```

Comment:
OH NOOOOOOOO.  We don't want any types of ```any``` here.  Check out what type it is and replace the ```any```.  I would assume it is a string? Maybe.  You can do ```console.log(dayjs(date).format('ddd. MMM DD, YYYY');```